corehq/apps/app_manager/views/download.py

```diff
@@ -143,7 +143,6 @@ def download_jad(request, domain, app_id):
     """
     app = request.app
     if not app.copy_of:
-        app.set_form_versions(None)
```

Comment:
FWIW, looking at https://github.com/dimagi/commcare-hq/commit/4066c8b793073c88c1d9badcdf445ec1ae505f1f and https://github.com/dimagi/commcare-hq/commit/cb6c114a88aec6806765402f11e867ab097c4a37 I agree that these can be removed.

Reply:
Can you clarify?  Do you mean that `set_media_versions` should also be removed?  Or just this call to `set_form_versions`?

Reply:
just `set_form_versions`

Reply:
cool, thanks for the confirmation